Microsoft.Toolkit.Uwp.UI.Media/Geometry/CanvasPathGeometry.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Numerics;
+using System.Text;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.Brushes;
+using Microsoft.Graphics.Canvas.Geometry;
+using Microsoft.Toolkit.Uwp.UI.Media.Geometry.Common;
+using Microsoft.Toolkit.Uwp.UI.Media.Geometry.Parsers;
+using Windows.UI;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry
+{
+    /// <summary>
+    /// Helper Class for creating Win2d objects
+    /// </summary>
+    public static class CanvasPathGeometry
```

Comment:
@vgromfeld @Sergio0694 in this case does the static matter? Just trying to learn what I should be watching for based on our work we did in #3168.

Reply:
Since this class contains only static methods, it will not be an issue. 
It will become problematic if we start adding a lot of static fields.